godtools/App/Features/SetupParallel/SetupParallelLanguageView.swift

```diff
@@ -6,19 +6,19 @@
 //  Copyright © 2021 Cru. All rights reserved.
 //
 
-import Foundation
 import UIKit
 
-class SetupParallelLanguageView: UIViewController {
+class SetupParallelLanguageView: UIViewController, UIGestureRecognizerDelegate {
```

Comment:
Hey @reldredge71 I have a functionality question on this one.  If I tap a language from the list, and then tap the overlays to close out the setup parallel language modal, my selection will be saved to settings.  I'm wondering if we only want to save that selection if the Get Started button is tapped.  In my head I was thinking tapping the overlay would cancel out, but maybe some people would want the option to stick.  Could you check with Ann and Daniel on this one and see when that setting should be saved.  If at the moment of tapping it in the list, or when Get Started is tapped.

Reply:
I can ask, but I think what I have is what's expected.

Reply:
Thanks Robert.

Reply:
Justin said that he would like the parallel language to be set even if the user doesn't press "Get Started".  Doing it the way I have it could also give the app a couple more seconds to pre-load the parallel languages for the first few tools.  I know that the Favorited tools will download the primary language automatically when landing on the Tools Flow, is the same true for parallel language?